docs/api/presenter.md

```diff
@@ -276,3 +276,22 @@ class HelloWorldPresenter extends Presenter {
   }
 }
 ```
+
+### getRepo(repo, props)
+
+Runs before assigning a repo to a Presenter. This method is given the
+parent repo, either passed in via `props` or `context`. By default, it
+returns a fork of that repo, or a new Microcosm if no repo is
+provided.
+
+This provides an opportunity to customize the repo behavior for a
+particular Presenter. For example, to circumvent the default Presenter
+forking behavior:
+
+```javascript
+class NoFork extends Presenter {
+  getRepo (repo) {
+    return repo
```

Comment:
This is the same repo given to the Presenter (unless no repo is given). Is that what you mean?